crates/wit-encoder/tests/functions.rs

```diff
@@ -1,88 +1,72 @@
 use pretty_assertions::assert_eq;
-use wit_encoder::{Params, Result_, Results, StandaloneFunc, Type};
+use wit_encoder::{
+    Interface, Package, PackageName, Params, Result_, Results, StandaloneFunc, Type,
+};
 
-const PACKAGE: &str = "package foo:functions;
-
-interface functions {
-    f1: func();
-    f2: func(a: u32);
-    f3: func() -> u32;
-    /// this is a documentation comment
-    /// for the f4 function
-    f4: func() -> tuple<u32, u32>;
-    f5: func(a: f32, b: f32) -> tuple<u32, u32>;
-    f6: func(a: option<u32>) -> result<u32, f32>;
-    f7: func() -> (u: u32, f: f32);
-    f8: func() -> (u: u32);
-    f9: func() -> result<f32>;
-    f10: func() -> result<_, f32>;
-    f11: func() -> result;
-}
-";
+const PACKAGE: &str = include_str!("./functions.wit");
```

Comment:
I personally find it nice when tests are as self-contained as they can be, e.g. "everything in one file", so could the WIT literal be inlined in these files instead of using `include_str!`?

Reply:
Yes, absolutely! Are you okay if I use [indoc](https://github.com/dtolnay/indoc) to make sure the formatting is preserved when inlining?

Reply:
sure!